pypdf/_writer.py

```diff
@@ -2874,6 +2875,80 @@ def reset_translation(
         else:
             raise Exception("invalid parameter {reader}")
 
+    def set_page_label(
```

Comment:
Please add a docstring explaining what this function does / what the parameters mean.